CaesarCipher.java

```diff
@@ -0,0 +1,71 @@
+/*
```

Comment:
Hey, thanks for sharing your code. Here are the JUnit test cases, and their outcomes grouped by their result.
Please see line comments for suggested changes.

### Pass
assertEquals("some text", week1.Encode("0:some text"));
assertEquals("tpnf ufyu", week1.Encode("1:some text"));
assertEquals("222222", week1.Encode("1:111111"));
assertEquals("bbbbbb", week1.Encode("1:aaaaaa"));
assertEquals("AAAAA", week1.Encode("1:ZZZZZ"));
assertEquals("00000", week1.Encode("1:99999"));
assertEquals("99999", week1.Encode("-1:00000"));
assertEquals("AAAAA", week1.Encode("-1:BBBBB"));

### Fail
assertEquals("$$$$$", week1.Encode("1:$$$$$"));
assertEquals("00000", week1.Encode("-1:11111"));
assertEquals("$$$$$", week1.Encode("-1:$$$$$"));
assertEquals("ZZZZZ", week1.Encode("-1:AAAAA"));